src/Fakes/v1/YousignFaker.php

```diff
@@ -117,24 +117,28 @@ public function __construct()
                     Response::HTTP_CREATED
                 ),
 
-                $url . Yousign::SIGNATURE_REQUESTS_BASE_URL . '/*/' . Yousign::DOWNLOAD_DOCUMENT_URL . '/*' => Http::response(
+                $url . Yousign::SIGNATURE_REQUESTS_BASE_URL . '/*/' . Yousign::DOCUMENT_URL . '/*/' . Yousign::DOWNLOAD_URL => Http::response(
                     self::FAKE_DOCUMENT_CONTENT,
+                    Response::HTTP_OK
```

Comment:
@Martab93 perché li hai aggiunti, se sono il default?
Avevo tolto io dove necessario perché c'era created dappertutto, e invece non è vero che ti da sempre 201. In alcuni casi ti da 201 ed in altri 200, quindi per essere più fedele al funzionamento reale di YS avevo cambiato.

Reply:
non ho fatto caso al fatto che fossero il default, sry, comunque ho controllato per le chiamate sulla documentazione quale fosse quello corretto

Reply:
È il default, phpstorm dovrebbe anche segnalartelo che stai specificando un valore che coincide con il default.

Alla prossima iterazione togliamoli.

cc @Pingus74